### ISSUE TITLE
Sound completely doesn’t sync

### ISSUE THREAD

**Issue by chief18** [Gericom/GBARunner2#172]
When I play Pokémon Fire Red (now I’m on the italian version) the game is fully playable, but the sound isn’t synced at all.

**Comment by RocketRobz** [comment]
I assume this is being run on a flashcard?

**Comment by chief18** [comment]
> I assume this is being run on a flashcard?

Yes it is... is it the reason why?

**Comment by chief18** [comment]
?

**Comment by zenseii** [comment]
Yes, currently many games have sound desync unless they're run from gbarunner2 on an unlaunched dsi's or 3ds' sd card. If you don't have the option to run it on either one of those two then there's not much you can do.

Also you should check the gbarunner2 wiki to know what to expect in terms of compatibility from the backup you're trying to run. You'll find it under compatibility list on the front page of this repo.
